ui/main.qml

```diff
@@ -220,7 +220,7 @@ ApplicationWindow {
     SystemTrayIcon {
         id: systemTray
         visible: true
-        icon.source: chatsModel.unreadMessagesCount > 0 ? "shared/img/status-logo-new.png" : "shared/img/status-logo.png"
+        icon.source: "shared/img/status-logo.png"
```

Comment:
There was no new equivalent for a menu bar icon when there are unread messages, so I went with just one icon for now.

@simonricoo lemme know if there's such a thing and this needs update

Reply:
Yes, dont change the appearance of the menu bar icon when there is a notification or new message or anything. Use only app icon to display badges - thats the normal behaviour on MacOS
<img width="180" alt="Screenshot 2021-01-14 at 13 21 30" src="https://user-images.githubusercontent.com/52249767/104590317-70ddb700-566b-11eb-858b-cf3c9558edc9.png">

Reply:
I might be wrong here, but I wouldn't be surprised if we can't update the System app icon at runtime.
That icon there has to be put in a certain location on your machine for macOS to render it. That "putting it in that location" happens at compilation of the app. 

Not sure we can do anything about that later on. Maybe @michaelsbradleyjr has an idea

Reply:
You added the file called `status-icon.icns`, but it's not the one you use here.
Also, is there a way to have the red dot on new messages, it's very helpful. This is just the TrayIcon BTW. I'm not sure where it appears on Mac, but on Windows, it's the one small one in the bottom right corner and on Linux, it's also a small one in the top right, so it might not be what you want.

Reply:
> You added the file called status-icon.icns, but it's not the one you use here.
That's correct. The `status-icon.ics` is used by the app as App icon and will be put into a particular location when the app is installed. (That's the one only used on mac, check this code here: https://github.com/status-im/nim-status-client/blob/master/src/nim_status_client.nim#L45-L57)

The one used there (`status-logo.png`), which I've updated, is the one used in the menu bar. 
> Anyway, the point stands that changing this icon here will modify not only Mac
Correct.

Reply:
Ah of course. I had assumed that you added a new icon and forgot to use it and didn't see it was the same name as before. I blame Github for showing it as `+99KB` and no `-X`. It really made it look like it's just a new file.

Anyway, it would be nice to still have the icon in the Tray have the update icon just like Discord. It's a very simple feature but it helps me a lot to know if I have new messages.

Example for discord:
![image](https://user-images.githubusercontent.com/11926403/104739937-56f8ae80-5715-11eb-8f8d-bbda0d675efc.png)

We can add it in another PR I guess. Is that something that's good with you @simonricoo